### ISSUE TITLE
"Show All Tabs" keyboard shortcut doesn't do anything

### ISSUE THREAD

**Issue by hachimagi** [zen-browser/desktop#1969]
### Captchas

- [X] I have read the instructions.
- [X] I have searched existing issues and avoided creating duplicates.
- [X] I am not filing an enhancement request.

### What happened?

by default, it is bound to "Ctrl+Shift+Tab", which gets overwritten by firefox's default binding, making it switch to previous tab.
even if rebinded to a shortcut that doesn't get overwritten, it just doesn't seem to do anything.

### Reproducible?

- [X] I have checked that this issue cannot be reproduced on Mozilla Firefox.

### Version

1.0.1-a7

### What platform are you seeing the problem on?

Linux

**Comment by vinibortoletto** [comment]
I can confirm it doesn't do anything even after changing the keys binded to it.

Also, I think a better name for this shortcut would be "Expand Vertial Tabs", at least that's what I except this shortcut to do.

**Comment by jasiralavibiztripz** [comment]
It would be great if the tabs can be listed for a quick search.
With option for the user to show tabs from all windows, current window or just current workspace.

Something like: https://addons.mozilla.org/en-US/firefox/addon/tab_search/
Should be default. The UI could be improved by placing it in the center for better focus and many be showing more lines.

**Comment by jasiralavibiztripz** [comment]
We could even make use of the address bar for this.
I know that there is the % (+space) combination that filters and shows just the tabs. 
But it's too long: Ctrl + L to focus on the address bar, then Shift+5+ space to activate the (%) tabs filter.
We should be able to achieve it easily with Alt +A. Same for others like Bookmarks and History.
e.g. 
Alt+F→ Find / Show all tabs (%)
Alt+B → Search in Bookmarks (^)
Alt+H → Search in History (*)

I liked the color coding and other details in the https://addons.mozilla.org/en-US/firefox/addon/tab_search/ extension. could be integrated into the zen browsers default features.

**Comment by salamijason** [comment]
Show All Tabs does nothing on this end too. Thank you for pointing it out!

**Comment by jasiralavibiztripz** [comment]
Hi, I can see that the "Ctrl+Shift+Tab" now works, but we are still unable to change the shortcut binding. 

Personally I prefer "Alt+A" for this. 

It's better to change the shortcut away from "Ctrl+Shift+Tab" since this is (as in Firefox) expected to be the reverse of "Ctrl+Tab" (change to next tab).

**Comment by mr-cheffy** [comment]
I didnt even know that shortcut existed...

**Comment by mdj-uk** [comment]
For me the default shortcut (ctrl+shift+tab) still does nothing - unlike the comments above.  I'm on version 1.7.2b (Firefox 134.0.2) (64-bit) with MacOS 14.2.1

It also does nothing when rebound, e.g. to ctrl+cmd+a.

**Comment by fahimaloy** [comment]
Hello,
I have been using zen browser in my Fedora 41 KDE Plasma (6.3.2)
The Zen Browser Version is 1.9b (Firefox 136.0) (64-bit)

![Image](https://github.com/user-attachments/assets/e92da187-b4f7-4a59-853e-9e725de49637)

Here is the issue,
by default, The Show All Tabs Shortcut was set to Ctrl+Shift+T
But If I pressed that, It would only switch tabs in reverse (Like most other browsers does)
then I tried to switching it to something else, the new keybinding does absolutely nothing

N.B: I am using flatpak version

**Comment by salamijason** [comment]
Yep it's still in the shortcuts menu but doesn't do anything. prolly best to just remove it @mauro-balades

**Comment by mr-cheffy** [comment]
Hmmm alr!

**Comment by fahimaloy** [comment]
@mauro-balades 
I really need this feature, (like how arc has right now after pressing ctrl+tab)
is there any workaround for this? or is this under development or not considered in zen at all?

![Image](https://github.com/user-attachments/assets/43215360-3244-40f5-b826-4a1253101fbc)

**Comment by halfmonty** [comment]
@fahimaloy  The behavior is a bit odd. The "show all tabs" shortcut still seems to do nothing. However, if under Settings -> General, you check the box for "Ctrl+Tab cycles through tabs in recently used order" then you get that sort've arc view of the last 7 or so tabs you've accessed. So maybe that'd make you happy. 

However this seem to override / remove the built in Ctrl-Tab and Shift-Ctrl-Tab default firefox bindings for switching tabs in sequential order without a UI popup. So you get either the default next/previous tab switch that allows you to navigate through ALL tabs or if you toggle on the "recently used order" under general then you lose that ability to navigate all tabs and gain the fancy arrow-navigable arc-like preview popup thing but can only navigate the last 7 tabs.

I guess what I would have expected by checking that box is the ctrl-tab and shift-ctrl-tab would work the same but just move through tabs in recently used order instead of tab order, I was not expecting a new popup UI thing or being limited to the last 7.

So imo for clarity, the shortcut should probably work and be the thing that brings up this fancy arc-like UI and the general toggle should just change order that tabs cycle through. Those are two very separate things and it being behind a not very clearly labeled check box is confusing. But I think this behavior is also found in raw firefox so I don't think it's for the zen devs to fix.

I guess for now I'll use the "recently used order" toggle and use the % search to navigate through all tabs or Ctrl-Page Up / Down to got to tab up/down.

**Comment by Byte-Entropy** [comment]
Hey! Coul anyone solve the Issue of remapping the shortcut? For me it worked right after installation but after switching it to 
'ctrl + alt + A' it stopped owrking. I added then the "List all tabs" option from the customization menu and it doesn't work either..
